crates/starknet-types-core/src/curve/projective_point.rs

```diff
@@ -0,0 +1,212 @@
+use crate::curve::affine_point::AffinePoint;
+use crate::curve::curve_errors::CurveError;
+use crate::felt::Felt;
+use core::ops;
+use lambdaworks_math::cyclic_group::IsGroup;
+use lambdaworks_math::elliptic_curve::short_weierstrass::curves::stark_curve::StarkCurve;
+use lambdaworks_math::elliptic_curve::short_weierstrass::point::ShortWeierstrassProjectivePoint;
+use lambdaworks_math::elliptic_curve::traits::EllipticCurveError::InvalidPoint;
+use lambdaworks_math::unsigned_integer::traits::IsUnsignedInteger;
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct ProjectivePoint(pub(crate) ShortWeierstrassProjectivePoint<StarkCurve>);
+
+impl ProjectivePoint {
+    pub fn new(x: Felt, y: Felt, z: Felt) -> ProjectivePoint {
+        Self(ShortWeierstrassProjectivePoint::new([x.0, y.0, z.0]))
+    }
+
+    /// The point at infinity.
+    pub fn identity() -> ProjectivePoint {
+        Self(ShortWeierstrassProjectivePoint::neutral_element())
+    }
+
+    /// Creates the same point in affine coordinates. That is,
+    /// returns [x / z: y / z: 1] where `self` is [x: y: z].
```

Comment:
`[x / z: y / z: 1]`

I don't understand this notation.
Do the brackets mean an array?
Does `/` mean division or separation? 
Does imply an operation, an equality or is it just a separation? 

Maybe it's standard and I'm just not well-versed enough in mathematics?

Reply:
It means division. That with the point (x, y, z) we create the new point
(x * inv_z, y * inv_z, 1)

where z_inv = multiplicative inverse of z

Reply:
I made some modifications in the documentation so it is clearer 
[971bef6](https://github.com/starknet-io/types-rs/pull/8/commits/971bef699bf35419fa7bb5ea55d12fe1b8958000)